policyengine_us/variables/gov/states/mi/tax/income/credits/homestead_property_tax/mi_total_homestead_property_tax_credit.py

```diff
@@ -0,0 +1,74 @@
+from policyengine_us.model_api import *
+
+
+class mi_total_homestead_property_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan total homestead property tax credit before the phase-out"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-508",
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/MI-1040CR.pdf#page=2",
+    )
+    defined_for = "mi_homestead_property_tax_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.credits.homestead_property_tax
+        p2 = parameters(period).gov.states.mi.tax.income
+
+        total_household_resources = tax_unit("mi_household_resources", period)
+        exemption_amount = tax_unit(
+            "mi_homestead_property_tax_credit_household_resource_exemption",
+            period,
+        )
+
+        # seniors
+        # SECTION A: SENIOR CLAIMANTS (if you checked only box 5a)
+        older_spouse_age_eligible = (
```

Comment:
use the new variable here, and anywhere else this is defined (may be outside this credit)